index.html

```diff
@@ -1235,53 +1235,35 @@
   uses</a> for ongoing discussion and iteration on this principle.
 </aside>
 
-In maintaining duties of [=duty of protection|protection=], [=duty of discretion|discretion=] and [=duty of loyalty|loyalty=], user
-agents should be conservative when sharing data that isn't needed to satisfy a user's immediate goals.
-
-On the Web, user agents and sites often communicate data outside what is needed to load and display a
-page. These <dfn data-lt="ancillary use">ancillary uses</dfn> include:
-
-* browser telemetry, including:
```

Comment:
Pete will move these into "example" and use just the highest level in the list

Reply:
I've added the top level examples back into an "example" block, and moved it directly after our informal definitions of ancillary data and ancillary uses of data